:x: There was a problem during the validation job.

Someone from the @docker/extensions will have to take a look at this issue.
We will let you know in this issue when it is fixed.

---
See [action]({{ .workflow_url }}) for more details.
